web-local/config-stashtest.yaml

```diff
@@ -0,0 +1,269 @@
+# Select service mode - See EmoServiceMode class to see all available options
+# This allows you to split Emo's micro services deployed separately. The default is STANDARD_ALL
+serviceMode: STASH
```

Comment:
This file, `config-stashmanagertest.yaml` and `config-stashworkertest.yaml` should all be identical except for the "serviceMode" attribute.